components/src/core/info-list-item/info-list-item.tsx

```diff
@@ -333,7 +333,7 @@ export const InfoListItem: React.FC<InfoListItemProps> = (props) => {
                     : Colors.black[500]
                 : Colors.white[50]; // default avatar is dark gray -> white text
         }
-        return statusColor ? statusColor : theme.colors.text;
+        return statusColor ? statusColor : theme.dark ? Colors.black[200] : Colors.gray[500];
```

Comment:
these should be coming from the updated theme properties and will need a fallback in the event that someone is incorrectly using an older theme with newer components, something like:
```suggestion
        return statusColor ? statusColor : (theme.dark ? theme.colors.textPalette.secondary || theme.colors.text);
```

Reply:
Let's leave a comment there so that we know what this logic is for